### ISSUE TITLE
[Bug]: Release 2.1.3 cannot install GPTK dependencies

### ISSUE THREAD

**Issue by JinchuanL** [Whisky-App/Whisky#624]
### Description

System: MacOS Sonoma 14.1.1
Machine: Macbook Pro 2020 M1 (8+256)
Whisky Version: 2.1.3

When you are trying to open Whisky without GPTK installed, it will ask you to install GPTK dependencies and it will stuck at the downloading page without any progress. No downloading in progress, no speed. Can't quit Whisky, open logs folder does not work, only force quit works.

### Steps to reproduce

1. Install Rosetta 2
2. Download Whisky 2.1.3 and unzip it, drag it to the Application
3. Open Whisky and click Next to install missing dependencies (GPTK)
4. Stuck at downloading dependencies page, can't quit Whisky but only force quit.

### Expected behaviour

It should install GPTK and proceed to the main page of Whisky.

### Logs

```bash
Open Logs Folder does not work: )
```

### What version of Whisky are you using?

2.1.3

### What version of macOS are you using?

Sonoma (macOS 14)

### Issue Language

- [X] Yes my issue is written in English

**Comment by JinchuanL** [comment]
I tried to download the earlier version which is 2.0.1 and it works as expected, finished downloading the GPTK dependencies but still, open Logs Folder does not work.

**Comment by IsaacMarovitz** [comment]
What happens if you try to access https://data.getwhisky.app/Libraries.zip in your browser?

**Comment by JinchuanL** [comment]
Unable to connect

Your connection is not private

data.getwhisky.app normally uses encryption to protect your information. When Chrome tried to connect to data.getwhisky.app this time, the website sent back unusual and incorrect credentials. This may happen when an attacker is trying to pretend to be data.getwhisky.app or a Wi-Fi sign-in screen has interrupted the connection. Your information is still secure because Chrome stopped the connection before any data was exchanged.

You cannot visit data.getwhisky.app right now because the website uses HSTS. Network errors and attacks are usually temporary, so this page will probably work later.

**Comment by JinchuanL** [comment]
> What happens if you try to access https://data.getwhisky.app/Libraries.zip in your browser?

Unable to connect

Your connection is not private

data.getwhisky.app normally uses encryption to protect your information. When Chrome tried to connect to data.getwhisky.app this time, the website sent back unusual and incorrect credentials. This may happen when an attacker is trying to pretend to be data.getwhisky.app or a Wi-Fi sign-in screen has interrupted the connection. Your information is still secure because Chrome stopped the connection before any data was exchanged.

You cannot visit data.getwhisky.app right now because the website uses HSTS. Network errors and attacks are usually temporary, so this page will probably work later.

**Comment by IsaacMarovitz** [comment]
You need working HTTPS to connect to the bucket.

**Comment by brianashe** [comment]
Possibly useful: this happened to me. I was stuck at "Downloading GPTK: Zero KB of Zero KB". I left it open, did a search, and wound up here. Meanwhile, Whiskey crashed in the background and I ignored it. Using a browser, I tried visiting https://data.getwhisky.app/ (got a 404, which is fine) and then https://data.getwhisky.app/Libraries.zip which downloaded the file. I launched Whiskey again and it worked right away. Could all be a coincidence, or maybe there is something that makes Whiskey not able to be the first app on a system to connect to data.getwhisky.app . This was on a freshly-updated Mac running Sonoma 14.2.1 and Whiskey 2.2.3.
